### PR TITLE
Remove snapshots of merged connectors

### PR DESCRIPTION
**Pull request by SkNuwanTissera**
# Description

Fixes https://github.com/ballerina-platform/ballerina-extended-library/issues/134
Fixes https://github.com/ballerina-platform/ballerina-extended-library/issues/128

Related  https://github.com/ballerina-platform/ballerinax-openapi-connectors/pull/546

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

**Test Configuration**:
* Ballerina Version: Ballerina Swan Lake Beta 4
* Operating System: Ubuntu
* Java SDK: Java 11

### Security checks
 - [X] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? 
 - [X] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: openapi/azure.sqldb/Ballerina.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
 org = "ballerinax"
 name = "azure.sqldb"
-version = "1.0.0-SNAPSHOT"
+version = "1.0.1"
```
Comment (LakshanSS) [inline openapi/azure.sqldb/Ballerina.toml:4]: This connector is not released yet.
So the version should be `1.0.0`

**Review by LakshanSS — CHANGES_REQUESTED** [review]
As these connectors are newly added ones, they havent been released yet to central.

So, their version should be `1.0.0`

File: openapi/azure.sqldb/Ballerina.toml (hunk 1 above)
Comment (SkNuwanTissera) [reply openapi/azure.sqldb/Ballerina.toml:4]: Sure. I will update as that.

Comment (SkNuwanTissera) [reply openapi/azure.sqldb/Ballerina.toml]: Updated on all places @LakshanSS

**Review by LakshanSS — APPROVED** [review]

**Pull request merged**